catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/TransformersCommandTest.java

```diff
@@ -69,13 +70,39 @@ public void testNullIds() throws Exception {
     when(bundleContext.getServiceReferences(InputTransformer.class, "(id=*)"))
         .thenReturn(serviceReferencesFiltered);
 
+    transformersCommand.inputOption = true;
```

Comment:
Add getter/setters for the options and use those here instead of directly accessing the field.